intro.md

```diff
@@ -1,13 +1,13 @@
 # Introduction
 
-I have started, lead and maintained many projects over several decades, out of
-which the most known one is probably [the cURL project](https://curl.se) I
+I have started, led and maintained many projects over several decades, out of
+which the most known one is probably [the cURL project](https://curl.se). I
```

Comment:
If that should be a period and a new sentence, then the next one should probably be improved I think. Maybe to add "it" in it like *I founded and have been leading **it** since 1998" ?

Reply:
I apologize; it seems I actually misread that part. I think that sentence is just fine. I submitted the pull request far too late at night I suppose.